pandas/tests/test_index.py

```diff
@@ -396,6 +396,66 @@ def test_symmetric_diff(self):
                 with tm.assertRaisesRegexp(TypeError, msg):
                     result = first.sym_diff([1, 2, 3])
 
+    def test_equals_op(self):
+        # GH9947, GH10637
+        index_a = self.create_index()
+        if isinstance(index_a, PeriodIndex):
```

Comment:
ok, another way is to override this method in the TestPeriodIndex, just FYI (not sure which is better though)